hook/mutate_pod.go

```diff
@@ -169,9 +182,19 @@ func (m podMutator) requestedPVCCapacity(ctx context.Context, pod *corev1.Pod, t
 				requested = ((req.Value()-1)>>30 + 1) << 30
 			}
 		}
+		dc, ok := sc.Parameters[topolvm.DeviceClassKey]
+		if !ok {
+			dc = topolvm.DefaultDeviceClassAnnotationName
+		}
+
+		total, ok := capacities[dc]
+		if !ok {
+			total = 0
+		}
 		total += requested
+		capacities[dc] = total
```

Comment:
Can this be simplified as follows?
```suggestion
		capacities[dc] += requested
```